packages/near-membrane-base/src/intrinsics.ts

```diff
@@ -122,6 +121,10 @@ const ReflectiveIntrinsicObjectNames = [
     'eval',
 ];
 
+const ESGlobalsAndReflectiveInstrinsicObjectNames = ESGlobalKeys.slice(0).concat(
```

Comment:
We can spread those in `[...ESGlobalKeys, ... ReflectiveIntrinsicObjectNames]`

Reply:
That's what I originally wrote, but for some reason I thought we were avoiding spread? Maybe I'm misremembering?

Reply:
Normally we can't but since this is initialization code (run before others we can. I think? Else we need to avoid `.` prototypal access. Can be ironed out in a follow-up.